bindings/rust/build.rs

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For any questions about this software or licensing,
+ * please email opensource@seagate.com or cortx-questions@seagate.com.
+ *
+ * Original author: Meng Wang <mengwanguc@gmail.com>
+ * Original creation date: 25-Jul-2022
+ */
+
+extern crate bindgen;
+extern crate run_script;
+
+use std::env;
+use std::path::PathBuf;
+use std::process::Command;
+use run_script::ScriptOptions;
+
+fn main() {
+    println!("cargo:rustc-link-search=all=/root/cortx-motr/motr/.libs/");
```

Comment:
I think we can define a variable to represent the TOP LEVEL source dir.
So, users can change it according to their own environment.